README.md

```diff
@@ -38,6 +38,7 @@ It does very primitive _'caching'_: aggregates all metrics and flushes them in r
 | -k | Keystone key | |
 | --auth-url | Keystone token URL | |
 | --config | Set options from a config file | |
+| --overwrite_collection_timestamp | Replace metric collection timestamp with ingest timestamp | False |
```

Comment:
Can this be passed in through the config file?  Does https://github.com/rackerlabs/blueflood-carbon-forwarder/blob/9270aee430ea59cad0a0ae78a7bc78b7c357637d/twisted/plugins/graphite_blueflood_plugin.py#L21 need to be updated for this?